### PR TITLE
Misha Herscu JS Reference Types Practice

### PR DESCRIPTION
**Pull request by MishaHerscu**
I haven't done the bonus section yet, but I still might.

### CODE REVIEW

**Comment by MishaHerscu** [comment]
"node filename.js" should work for both scripts now at the command line.

File: lib/mbta.js [hunk 1]
```diff
@@ -2,10 +2,43 @@
...
 
-const stopsBetweenStations = (startLine, startStation, endLine, endStation) => {
+// Sets up MBTA lines
+let redLine = ['South Station', 'Park Street', 'Kendall', 'Central'];
+redLine = redLine.concat(['Harvard', 'Porter', 'Davis', 'Alewife']);
+let greenLine = ['Government Center', 'Park Street', 'Boylston', 'Arlington'];
+greenLine = greenLine.concat(['Copley', 'Hynes', 'Kenmore']);
+let orangeLine = ['North Station', 'Haymarket', 'Park Street', 'State'];
+orangeLine = orangeLine.concat(['Downtown Crossing', 'Chinatown']);
+orangeLine = orangeLine.concat(['Back Bay', 'Forest Hills']);
```
Comment (RDegnen) [inline lib/mbta.js]: are you concatenating these to avoid linter errors?

File: lib/mbtaBonus.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+// Test should equal 7
+let redRedTest = bonusStopsBetweenStations('Red', 'Davis', 'Red', 'South Station', intersectionsList);
+console.log('redRedTest (Should equal 6): \nFinal Answer:', redRedTest);
+console.log('___________________________________');
+
+module.exports = {
+  bonusStopsBetweenStations,
+  stretch: false,
+};
```
Comment (RDegnen) [inline lib/mbtaBonus.js:89]: It all looks really good! Nice job.

Comment (MishaHerscu) [inline lib/mbtaBonus.js:89]: Thanks

File: lib/mbta.js (hunk 1 above)
Comment (MishaHerscu) [inline lib/mbta.js]: I was. In later version i just used commas and went across lines that way which was also fine.

**Pull request closed without merging**